tests/benchq/problem_ingestion/test_molecule_instance_generation.py

```diff
@@ -6,12 +6,17 @@
 )
 
 
+
+avas_hydrogen_chain_instance = generate_hydrogen_chain_instance(2)
```

Comment:
Could you encapsulate it in:
```
def _generate_avas_hydrogen_chain_instance(n_hydrogens):
    avas_hydrogen_chain_instance = generate_hydrogen_chain_instance(n_hydrogens)
    avas_hydrogen_chain_instance.avas_atomic_orbitals = ["H 1s"]
    avas_hydrogen_chain_instance.avas_minao = "sto-3g"
    return avas_hydrogen_chain_instance
```
Or sth like that?
I got a bit confused by what's going on here.

Reply:
Fixed in a2216750a1c9042466a4c39aff99db5299e1afe9. This is just creating one of the application instances to be used in the following test

Reply:
I have figured it, but it took me half a minute or so ;)
Thanks!